.builds/freebsd.yml

```diff
@@ -0,0 +1,20 @@
+image: freebsd/latest
+packages:
+  - basu
```

Comment:
I assume we'll have to build it ourselves for now, [like we do on alpine](https://github.com/emersion/mako/blob/master/.builds/alpine.yml).

Reply:
Could use a subproject if it's simpler, but last time I tried basu wouldn't build as a subproject.

Reply:
If basu API is stable there's not much point *not* to use system package.

Reply:
Yup, if it's available using the system package is better indeed. I wasn't sure whether it would just be in ports or also in pkg repos, and it now seems like it's the latter. Thanks for pushing it forward!